src/consts.ts

```diff
@@ -3,4 +3,5 @@ export default {
   localPath: process.env.STUDIO_YAML_DIR || "./replicated",
   upstreamEndpoint: process.env.STUDIO_UPSTREAM_BASE_URL || "https://api.replicated.com/market/",
   lintThreshold: process.env.STUDIO_LINT_THRESHOLD || "error",
+  lintRequired: process.env.STUDIO_LINT_REQUIRED === "true",
```

Comment:
why not support other things like `1` or `yes`? I feel like a lot of "flag-style" env vars support using `1` to mean true